config/openshift/v4_8_exp/translate.go

```diff
@@ -121,3 +131,31 @@ func ToConfigBytes(input []byte, options common.TranslateBytesOptions) ([]byte,
 		return cutil.TranslateBytesYAML(input, &Config{}, "ToMachineConfig4_8", options)
 	}
 }
+
+func addLuksFipsOptions(mc *result.MachineConfig) translate.TranslationSet {
+	ts := translate.NewTranslationSet("yaml", "json")
+	if !util.IsTrue(mc.Spec.FIPS) {
+		return ts
+	}
+
+OUTER:
+	for i := range mc.Spec.Config.Storage.Luks {
+		luks := &mc.Spec.Config.Storage.Luks[i]
+		// Only add options if the user hasn't already specified
+		// fipsCipherOption.  Do this in-place, since config merging
+		// doesn't support conditional logic.
+		for _, option := range luks.Options {
+			if option == fipsCipherOption {
```

Comment:
Should this be changed out to `strings.HasPrefix(option, fipsCipherOption)`? I think `--cipher=my-cipher-here` should be valid.

Reply:
Whoops, good catch.